README.md

```diff
@@ -1,2 +1,100 @@
 # Swarm_Robotics
-Srishti 2020
+## Srishti 2020
+<p align ="justify">
```

Comment:
@Pasa1912 Remove this `<p>` tag. Use this tag separately for individual paragraphs which you want to justify